### DISCUSSION TITLE
Backticks are visibly shown in the rendered markdown for inline code blocks

### DISCUSSION THREAD

**Discussion by rotemdan** [open-webui/open-webui#4650] [General]
In one of the recent updates (I'm currently using `v0.3.13`), inline code blocks have started to show with visible backticks.

For example:
![Screenshot_1](https://github.com/user-attachments/assets/b0d11ccb-9b30-4322-84ad-9796498f8dd1)

Example rendered HTML:
```html
<li>We initialize the <code class="codespan">`maxElement`</code> variable to the first element of the array, and the <code class="codespan">`maxValue`</code> variable to the value extracted by the selector function for that element.</li>
```

**Comment by tjbck** [comment]
Fixed on dev.

**Comment by rotemdan** [comment]
Thanks.
